sound/soc/intel/boards/sdw_rt700.c

```diff
@@ -105,6 +106,53 @@ static int card_late_probe(struct snd_soc_card *card)
 }
 #endif
 
+static struct snd_soc_jack_pin sdw_jack_pins[] = {
+	{
+		.pin    = "Headphones",
+		.mask   = SND_JACK_HEADPHONE,
+	},
+	{
+		.pin    = "AMIC",
+		.mask   = SND_JACK_MICROPHONE,
+	},
+};
+
+static int headset_init(struct snd_soc_pcm_runtime *rtd)
+{
+	struct mc_private *ctx = snd_soc_card_get_drvdata(rtd->card);
+	struct snd_soc_component *component = rtd->codec_dai->component;
```

Comment:
For reverse-xmas style, snd_soc_component could be the first declaration as it is the longest.

Reply:
updated